File: C_Program/circle.c
//wap to accept radius of a circle and print area and circ. of circle

#include<stdio.h>
void main() {
	int radius;
	float area,circ,pi=3.14;
	printf("enter the radius of circle: ");
	scanf("%d",&radius);
	area=pi*radius*radius;
	circ=2*pi*radius;
	printf("\n area of circle=%.2f",area);
	printf("\n circ of circle=%.2f",circ);
} 

File: Java_Programs/Circle.java
//wap to accept radius of a circle and print area and circ of circle.
import java.util.*;
class Circle
{
	public static void main(String []args)
	{
		int radius;
		float pi=3.14f;

		Scanner s = new Scanner(System.in);

		System.out.println("Enter radius of a circle: ");
		radius = s.nextInt();

		float area =(float)(3.14 * radius * radius);
		System.out.println("Area of circle = " + area);

		float circ = (float)(2 * 3.14 * radius);
		System.out.println("Circ of circle = " + circ);
	}
}